evm_arithmetization/src/fixed_recursive_verifier.rs

```diff
@@ -2001,6 +2076,9 @@ where
             anyhow::Error::msg("Invalid conversion when setting public values targets.")
         })?;
 
+        // TODO(sdeng): Set to false when this segment contains no Keccak operations.
+        root_inputs.set_bool_target(self.root.use_keccak_tables, true);
```

Comment:
same here

Reply:
This is addressed in #657
The main purpose of this PR is the circuit change; it does not alter any behavior of the recursion system.